dojson/utils.py

```diff
@@ -13,6 +13,25 @@
 import six
 
 
+class IgnoreKey(Exception):
+
+    """The corresponding key has been ignored."""
+
+    pass
+
+
+def ignore_value(f):
```

Comment:
Thus can we keep the original name `ignore_value `? In this case is the PR ready for integration?